views/contact/form.php

```diff
@@ -9,21 +9,13 @@
  * @var $field \Yiisoft\Form\Widget\Field
  */
 
+use App\Widget\FlashMessage;
 use Yiisoft\Form\Widget\Form;
 use Yiisoft\Html\Html;
-use Yiisoft\Yii\Bootstrap5\Alert;
-
-if (isset($sent)) {
-    echo Alert::widget()
-              ->options(['class' => !$form->hasErrors() ? 'alert-success' : 'alert-danger'])
-              ->body(
-                  $sent && !$form->hasErrors()
-                      ? 'Thanks to contact us, we\'ll get in touch with you as soon as possible.'
-                      : 'One or more values are incorrect'
-              );
-}
 ?>
 
+<?= FlashMessage::widget() ?>
```

Comment:
Should it be in layout?

Reply:
It's possible, not sure if desired.

Reply:
Now I added the title to the contact page. If Flash is in layout, it will appear above titles, which looks... wrong.